src/admin/python/lsst/qserv/admin/render.py

```diff
@@ -0,0 +1,77 @@
+# This file is part of qserv.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
```

Comment:
LSST Python Style guide at https://developer.lsst.io/python/style.html#docstrings-should-be-written-for-all-public-modules-functions-classes-and-methods requires that:

> Write docstrings for all public modules, functions, classes, and methods. 

This means that you may need to have a docstring right here to explain the purpose and use of this module (in addition to what you already have for each function/class/method downstream.

Otherwise, code readers would have to guess what this module with a very generic name `renderer` (BTW, why not call it `targs_proc`?)  is doing.

Reply:
a clearer name is a good idea. I think I like `render_targs` tho (?)

Reply:
👍